x/auth/keeper/params_test.go

```diff
@@ -1,92 +0,0 @@
-package keeper_test
-
-import (
-	"github.com/cosmos/cosmos-sdk/x/auth/types"
-)
-
-func (s *KeeperTestSuite) TestParams() {
```

Comment:
test is not needed as Get/Set is inherently already tested in collections and validation of params is tested in `types/params_test.go`.